src/skycoin/config.go

```diff
@@ -391,6 +391,7 @@ func (n *NodeConfig) applyConfigMode(configMode string) {
 	}
 }
 
+// nolint: unparam
```

Comment:
When the `nolint` directive is above the function, does it apply to the whole function?  Normally we put this directive at the end of the function signature line